src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -27,6 +27,14 @@
 public class ElasticsearchSinkConnectorConfig extends AbstractConfig {
 
   public static final String CONNECTION_URL_CONFIG = "connection.url";
+
+  public static final String CONNECTION_USERNAME_CONFIG = "connection.username";
+  public static final String CONNECTION_USERNAME_DOC =
+      "HTTP Basic Auth username (towards Elasticsearch)";
+  public static final String CONNECTION_PASSWORD_CONFIG = "connection.password";
+  public static final String CONNECTION_PASSWORD_DOC =
+      "HTTP Basic Auth password (towards Elasticsearch)";
```

Comment:
Maybe "Password used to authenticate with Elasticsearch. The default is null, and authentication will only be performed if a non-null username and password are specified."?